eli5/sklearn/permutation_importance.py

```diff
@@ -182,6 +191,9 @@ def fit(self, X, y, groups=None, **fit_params):
             self.estimator_ = clone(self.estimator)
             self.estimator_.fit(X, y, **fit_params)
 
+        if pandas_available and isinstance(X, pd.DataFrame):
+            self.scorer_ = self.wrap_scorer(self.scorer_, X.columns)
```

Comment:
I think it would be better to move all `self.scorer_` - related code close to each other - maybe move `self.scorer_ = check_scoring(self.estimator, scoring=self.scoring)` line just before this new code, so that they're not separated by `prefit` handling code?